changelog.md

```diff
@@ -3,6 +3,11 @@ This file contains all the notable changes done to the Ballerina JWT package thr
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.0-beta.1] - 2021-05-06
+
+### Changed
+- [Improve test coverage](https://github.com/ballerina-platform/ballerina-standard-library/issues/1010)
```

Comment:
IMO, this does not have to go to change log. As this not functional change but rather a QoS like change.

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-jwt/pull/203/commits/43eacc5f707bf3de329b21cc7ea4862e0951897d